src/include/storage/arrow_block_metadata.h

```diff
@@ -97,7 +97,7 @@ class ArrowVarlenColumn {
   /**
    * @return the offsets array
    */
-  uint32_t *Offsets() const { return offsets_; }
+  uint64_t *Offsets() const { return offsets_; }
```

Comment:
Why were all these changed to `uint64_t`

Reply:
Because the offsets array defined in arrow is 64-byte. If we use 32-byte in the system, we cannot directly use the offsets block.